repositories/wptuning.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
The file name should be wp-tuning.json to be consistent with the slug property.

Reply:
OK, :-(
I change the slug because it exist an another plugin : https://wordpress.org/plugins/wp-tuning/

Reply:
I change the slug because it exist an antther plugin with same slug : https://wordpress.org/plugins/wp-tuning/